src/js/menu/acquia_lift.personalize.views.js

```diff
@@ -1436,11 +1119,6 @@
       this.listenTo(this.model, 'change:isActive', this.onEditModeChange);
       this.listenTo(this.campaignCollection, 'change:isActive', this.onCampaignChange);
 
-      this.onPageVariationEditModeProxy = $.proxy(this.onPageVariationEditMode, this);
```

Comment:
That's the code that triggers it, I meant the user action to trigger it and show an unexpected behavior. Nevertheless, I am able to see some errors that you are seeing now, so I am bringing those code back as suggested.